_posts/2025-01-04-glorias-brink-epoch.md

```diff
@@ -0,0 +1,138 @@
+2022 to continue her work. That year she continued her P2P research, expanded
+the work to include fee bumping more generally, posted a discussion on RBF
+improvements to the [mailing
+list](https://gnusha.org/pi/bitcoindev/CAFXO6=LGbaur6XQrE+6a6mAAHXduOCXoWPTgPosxAG59ZkK6Gg@mail.gmail.com/),
+and posted a P2P protocol proposal which later became [BIP 331](https://github.com/bitcoin/bips/blob/master/bip-0331.mediawiki) to the [mailing
+list](https://gnusha.org/pi/bitcoindev/CAFXO6=JROe_9ih2h+_CCH-UbxehsM5RQ6YyNnPesEpveBEtdow@mail.gmail.com/)
+May 2022. That same year, she was named a Bitcoin Core maintainer by her peers,
+an honor that speaks volumes about her technical skill and collaborative spirit.
+
+In 2023, Brink continued support for Gloria and her work and, recognizing the
+valuable technical perspectives she has on the Bitcoin Core project, also
+invited her to join our grant committee to help evaluate funding applications.
+Meanwhile, Gloria continued her work on package relay and v3 transaction
+relay/TRUC, projects that would eventually encompass a whopping 53 pull requests
+over the years. She also worked to achieve buy-in on her projects from her
+peers, resulting in the package relay project making the top priority projects
+list for the Bitcoin Core project for three consecutive releases (26.0, 27.0,
+28.0).
+
+Much of Gloria’s work was included in the recent [Bitcoin Core 28.0
+release](https://bitcoinops.org/en/bitcoin-core-28-wallet-integration-guide/),
+including one parent one child (1P1C) relay, [BIP 431](https://github.com/bitcoin/bips/blob/master/bip-0431.mediawiki) TRUC transactions, and 1P1C
+package RBF. These features have garnered enthusiastic reception from Lightning
+engineers as well as other developers building on Bitcoin.
+
+## Beyond Mempool and Policy
+
+But Gloria’s work goes beyond the keystone projects that she helped drive
+forward. She has provided thousands of review comments on the Bitcoin Core
+GitHub repository over the years and authored dozens of other pull requests to
+improve the security and reliability of the codebase including wallet and P2P
+work. Her role as a Bitcoin Core maintainer also involves a myriad of thankless
+maintenance and administrative tasks that are important for the project,
+including fixing bugs and creating releases.
+
+Gloria’s thoughts on becoming a Bitcoin Core maintainer at the time:
+
+"*Being a maintainer means I'll... focus on the health of Bitcoin Core and help
+shoulder some of the responsibilities of the other maintainers... I'm relieved
+to find Brink understanding and supportive, not because of the "new title," but
+out of the recognition that the Bitcoin ecosystem is built on software that
+requires sustained, long-term effort to maintain, regardless of the coin's
+price.*"
+
+## Advocacy, Mentorship, and Education
+
+Beyond code, Gloria’s impact is felt in the broader Bitcoin community through
+her prolific advocacy and mentorship:
```

Comment:
If you're interested in a quote from 2023 self-eval:

The importance of advocacy and education is often understated. I limit this to 10-15% of my time, as there is an infinite supply of things to do. My framework is to allocate this time across a “funnel.” The top is spreading a message “the code needs work and runs on donations,” and the bottom might be recruiting somebody to work on Bitcoin Core and looking at PRs together.

Reply:
added!